src/mail.sh

```diff
@@ -135,6 +147,55 @@ function validate_email_list()
   return 0
 }
 
+# This function generates the appropriate recipients of the patches and saves
+# them in files to be read by the script passed to `git send-email`
+#
+# @kernel_root: The path to the root of the current kernel tree
+# @rev_list:    The list of revisions used to generate the patches
+#
+function generate_kernel_recipients()
+{
+  local kernel_root="$1"
+  local rev_list="$2"
+  local version="$3"
+  local to=''
+  local cc=''
+  local to_list=''
+  local cc_list=''
+  local patch_cache="${KW_CACHE_DIR}/patches"
+  local cover_letter_to="${patch_cache}/to/cover-letter"
+  local cover_letter_cc="${patch_cache}/cc/cover-letter"
```

Comment:
The curly braces are used to mark the beginning and end of the variable name, this is essential when trying to expand something like: `${my_var}_suffix` because it ensures bash will not try to expand `$my_var_suffix`. In this case it is not in fact needed, as a slash is not a valid character for a variable name, but I chose to use it here to make it clear what is the variable I'm expanding.